packages/shared/src/utils/helpers/__tests__/start-date.ts

```diff
@@ -20,8 +20,10 @@ describe('start_date', () => {
                 start_type: 'spot',
                 submarket: 'major_pairs',
                 underlying_symbol: 'frxAUDJPY',
+                forward_starting_options: [],
             };
-            expect(buildForwardStartingConfig(contract, {})).to.be.empty;
+            /* eslint-disable no-unused-expressions */
+            expect(buildForwardStartingConfig(contract, [])).to.be.empty;
```

Comment:
```suggestion
            expect(buildForwardStartingConfig(contract, [])).to.be.empty();
```

Reply:
@niloo-fs do we still need  /* eslint-disable no-unused-expressions */  here?🙂

Reply:
Yes we need that, if I don't add it I won't be able to commit my changes!